TACC/Core-Portal
# Bug/WP-782: Resized View Full Path link in modals

## Overview

The "View Full Path" links in modals is now too big compared to the rest of the text in the modal.

## Related

* [WP-782](https://tacc-main.atlassian.net/browse/WP-782)

## Changes

I added a `modalButton` style to the "View Full Path" links and used CSS to reduce its size.

## Testing

1. Log in to [cep.test/workbench](cep.test/workbench).
2. Select a file and click on either Move or Copy.
3. Note that "View Full Path" is now more appropriately-sized.

## UI

<details open><summary>As of a9c89af</summary>

| WP-782 | WP-50 |
| - | - |
| <img width="960" alt="PR 1036 WP-50" src="https://github.com/user-attachments/assets/64ef0ae8-10bd-41d1-b45f-86a434e15a72" /> | <img width="960" alt="PR 1036 WP-782" src="https://github.com/user-attachments/assets/8050581d-55fb-42c2-80bc-5a61f849d803" /> |

</details>
<details><summary>As of 7778ead</summary>

![image](https://github.com/user-attachments/assets/c62a86db-40b9-44f4-aaf0-18fe3b8e704a)

</details>

## Notes

@wesleyboar updated UI after a9c89af.
